tsdb/head.go

```diff
@@ -1757,7 +1784,10 @@ func newStripeSeries(stripeSize int, seriesCallback SeriesLifecycleCallback) *st
 		s.series[i] = map[chunks.HeadSeriesRef]*memSeries{}
 	}
 	for i := range s.hashes {
-		s.hashes[i] = seriesHashmap{}
+		s.hashes[i] = seriesHashmap{
+			unique:    map[uint64]*memSeries{},
+			conflicts: map[uint64][]*memSeries{},
```

Comment:
Because there are a small number of hash conflicts, is it possible to set conflicts to empty at the beginning?

Reply:
Do you mean `conflicts: nil` ? 
Yes; at the cost of checking and initializing it inside `set()`.  This would make the base memory smaller.